### ISSUE TITLE
Strange message logged to console

### ISSUE THREAD

**Issue by ddolcimascolo** [mjmlio/mjml#1857]
**Describe the bug**

The message
```
mjmlConfigOptions ------------------------------------------>
 {}
```
has started to pop all over the place in our builds. Can you advise ?

**To Reproduce**

The problem started a few versions back, I can't tell exactly when

**Expected behavior**

No message logged

**MJML environment (please complete the following information):**
 - OS: Linux, Ubuntu 18.04
 - MJML Version: 4.6.0, 4.6.1
 - NodeJS code

**Screenshots**

![image](https://user-images.githubusercontent.com/5468291/75882767-1ad09e00-5e22-11ea-935c-d459efd323c5.png)

**Comment by iRyusa** [comment]
Hi,

It's fixed in 4.6.2 as mentioned here #1856

**Comment by ddolcimascolo** [comment]
Thanks
